### PR TITLE
Sequantiality test

### PR DESCRIPTION
**Pull request by npmccallum**

### CODE REVIEW

File: .tests/formatting-sequentiality [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    if reduce(lambda x, y: x or y, map(lambda x: x.startswith('.'), dirs[1:]), False):
+        continue
+
+    # Append all files
+    rfcs += [(f, root) for f in files]
+
+status = 0
+for i, (name, root) in enumerate(sorted(rfcs)):
+    if i != int(name.split('-')[0]):
+        print("%s/%s is non-sequential!" % (root, name), file=sys.stderr)
```
Comment (lkatalin) [inline .tests/formatting-sequentiality]: Would it make sense to add which number it should be, in case someone is confused?

Comment (npmccallum) [reply .tests/formatting-sequentiality]: @lkatalin Fixed.

**Review by lkatalin — APPROVED** [review]
:+1:

**Review by mbestavros — APPROVED** [review]
LGTM!

**Review by mbestavros — APPROVED** [review]
:+1:

**Review by lkatalin — APPROVED** [review]

**Pull request merged**